doc/cephfs/cephfs-io-path.rst

```diff
@@ -0,0 +1,29 @@
+ Ceph File System IO Path
+=========================
+
+The Ceph File System (CephFS) stores file data as RADOS objects. File System
+clients access to RADOS directly for reading and writing file data. MDS only
+handle metadata operations.
+
+.. ditaa::
+            +---------------------+
+            |      CephFS FUSE    |
+            +---------------------+
+              |
+              V
```

Comment:
Instead of CephFS FUSE, maybe show the above as a generalized client. It could be ganesha or samba or some other application rather than FUSE.